src/contracts/misc/overview.md

```diff
@@ -0,0 +1,39 @@
+This document describes the roles and responsibilities of the Gho Stewards.
+
+## Gho Steward Contracts
```

Comment:
I think we can change it as the following, and move this file to `/docs/gho-stewards.md`
```suggestion
## Overview
```